### PR TITLE
Remove log logic

### PR DESCRIPTION
**Pull request by shargon**
Required for https://github.com/neo-project/neo/issues/1245
Require https://github.com/neo-project/neo/pull/1626

### CODE REVIEW

File: neo-cli/Logger.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+namespace Neo
+{
+    class Logger : Plugin, ILogPlugin
+    {
+        private static readonly object _lock = new object();
+
+        static Logger()
+        {
+            AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
```
Comment (erikzhang) [inline neo-cli/Logger.cs:13]: Maybe we can handle this event in neo core?

Comment (shargon) [reply neo-cli/Logger.cs:13]: save the errors in neo? I like it

Comment (shargon) [reply neo-cli/Logger.cs:13]: If we merge https://github.com/neo-project/neo/pull/1626 I will remove the logic from this PR

Comment (erikzhang) [reply neo-cli/Logger.cs:13]: No. I mean we handle this event in neo core, and send the message to the log stream. Then persist it in plugins.

Comment (erikzhang) [reply neo-cli/Logger.cs:13]: Then we can remove the `error.log`. The logs are unified.

**Pull request merged**